api/update_test.go

```diff
@@ -20,7 +21,7 @@ func TestUpdateHandlers(t *testing.T) {
 	cfg.Store.Type = "map"
 	cfg.Repo.Type = "mem"
 
-	inst, err := lib.NewInstance(tmpDir,
+	inst, err := lib.NewInstance(context.Background(), tmpDir,
```

Comment:
Move onto one line

Reply:
totally. I've got this fixed in what will be a follow-up PR